### ISSUE TITLE
is_json?

### ISSUE THREAD

**Issue by shmink** [michalmuskala/jason#64]
I've found myself in a situation where it would be nice to see if a value is JSON. I do get the error saying `invalid JSON ...` so I assume the functionality must be there. Perhaps there already is something in place in that fact.

For now, I've just tried to decode the value if it's successful then assume it is indeed JSON and valid.

**Comment by michalmuskala** [comment]
Yes, right now probably the best way to do this is to use `match?({:ok, _}, Jason.decode(data))`. The only way a specific function could be improved would be to actually implement a parser that does not produce the data - only traverses the input. It would most likely be noticeably faster, but it would probably also require reimplementing the entire parser without emitting data and I don't see myself doing that in the foreseeable future.
